application/contracts/pokemon_repository.go

```diff
@@ -0,0 +1,13 @@
+package contracts
+
+import "github.com/FernandoGal25/academy-go-q42021/domain/model"
+
+/*
+	Interface of pokemon repository, defines the methods of the repository needed to handle
+	the entity in question.
+*/
+type PokemonRepository interface {
```

Comment:
Exported elements should be documented with a brief description using `//` comments (as part of Go's best practices - https://go.dev/blog/godoc); please change this and all other occurrences of this specific issue all over the project.

```suggestion
// PokemonRepositare interface defines the methods of the repository needed to handle.
type PokemonRepository interface {
```